### PR TITLE
Ranking Criteria updates

### PR DESCRIPTION
**Pull request by pishifat**
catch change 1: https://osu.ppy.sh/community/forums/topics/901220
catch change 2: https://osu.ppy.sh/community/forums/topics/907104
multi mode spread requirements: https://osu.ppy.sh/community/forums/topics/845733
audio file quality clarifications: https://osu.ppy.sh/community/forums/topics/923648

---

### CODE REVIEW

File: wiki/Ranking_Criteria/en.md [hunk 1]
```diff
@@ -240,6 +233,8 @@ This category contains explicit allowance statements of concepts and rules that
 
 ### Guidelines
 
+- **The audio file of a song should have average peak frequencies of 16kHz or higher.** Lower values usually result in noticeably low audio quality and it is recommended to use a better `.mp3` unless none can be found. Average peak frequency can be shown in [audio spectogram generators](http://spek.cc/), which typically display 128kbps and 192kbps songs with average peak frequencies of 16kHz and 18kHz respectively.
```
Comment (cl8n) [inline wiki/Ranking_Criteria/en.md]:
```suggestion
- **The audio file of a song should have average peak frequencies of 16kHz or higher.** Lower values usually result in noticeably low audio quality and it is recommended to use a better audio file unless none can be found. Average peak frequency can be shown in [audio spectrogram generators](http://spek.cc/), which typically display 128kbps and 192kbps songs with average peak frequencies of 16kHz and 18kHz respectively.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 - **Hyperdashes of any kind are disallowed.** This is to ensure a manageable step in difficulty for novice players.
 - **Dashes may be used if the time between the ticks of the desired snapping is 125ms or higher.** As an example, 1/3 dashes would be allowed at 160 BPM and below, whereas 1/4 dashes would be allowed at 120 BPM and below.
 - **Edge dashes must not be used** as they require extremely precise timing which cannot be expected of less-experienced players.
-- **Basic dashes can't be used more than two times between consecutive fruits. If higher-snapped dashes are used, they must be used singularly (not in conjunction with other dashes).**
+- **Basic dashes can't be used more than two times between consecutive fruits.** If higher-snapped dashes are used, they must be used singularly (not in conjunction with other dashes).
```
Comment (cl8n) [inline wiki/Ranking_Criteria/osu!catch/en.md]:
```suggestion
- **Basic dashes must not be used more than two times between consecutive fruits.** If higher-snapped dashes are used, they must be used singularly (not in conjunction with other dashes).
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 3]
```diff
@@ -121,20 +121,20 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 
 - **Hyperdashes cannot be used on individual drops and/or slider repititions.** The accuracy and control required is unreasonable at this level and can create a situation where the player potentially fails to read the sliderpath.
 - **Hyperdashes may be used if the time between the ticks of the desired snapping is 125ms or higher.** As an example, 1/3 hyperdashes would be allowed at 160 BPM and below, whereas 1/4 hyperdashes would be allowed under 120 BPM.
-- **Basic hyperdashes can't be used more than two times between consecutive fruits. If higher-snapped hyperdashes are used, they must be used singularly (not in conjunction with other hyperdashes or dashes)**
+- **Basic hyperdashes can't be used more than two times between consecutive fruits.** If higher-snapped hyperdashes are used, they must be used singularly (not in conjunction with other hyperdashes or dashes).
```
Comment (cl8n) [inline wiki/Ranking_Criteria/osu!catch/en.md]:
```suggestion
- **Basic hyperdashes must not be used more than two times between consecutive fruits.** If higher-snapped hyperdashes are used, they must be used singularly (not in conjunction with other hyperdashes or dashes).
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 4]
```diff
@@ -121,20 +121,20 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
...
-- **Basic hyperdashes may be used in conjunction with antiflow patterns. If used, the spacing must not exceed a distance snap of 1.2x if followed by a walkable movement, or 1.1x the trigger distance if followed by a basic dash.**
+- **Basic hyperdashes may be used in conjunction with antiflow patterns.** If used, the spacing must not exceed a distance snap of 1.2 times the trigger distance when followed by a walkable movement, or 1.1 times the trigger distance when followed by a basic dash.
 - **Dashes may be used if the time between the ticks of the desired snapping is 62ms or higher.** As an example, 1/6 dashes would be allowed at 160 BPM and below, whereas 1/8 dashes would be allowed at 120 BPM and below.
-- **Basic dashes can't be used more than four times between consecutive fruits. Higher-snapped dashes can be used up to two times between consecutive fruits, provided there isn't a direction change between them.**
+- **Basic dashes can't be used more than four times between consecutive fruits.** Higher-snapped dashes can be used up to two times between consecutive fruits, provided there isn't a direction change between them.
```
Comment (cl8n) [inline wiki/Ranking_Criteria/osu!catch/en.md]:
```suggestion
- **Basic dashes must not be used more than four times between consecutive fruits.** Higher-snapped dashes can be used up to two times between consecutive fruits, provided there isn't a direction change between them.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 5]
```diff
@@ -180,7 +180,7 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 - Combos should not exceed 16 objects including slider tails and repeats. Spinners are an exception.
 - Note density should follow a mostly 1/2 + 1/4 and/or 1/3 + 1/6 pattern - 1/8 patterns and higher can be used sparingly.
 - 1/8 and higher hyperdashes should not be used between consecutive object pairs.
-- Edge dashes may be used with caution for a maximum of three consecutive objects, and should not be used in conjunction with hyperdashes.
+- Edge dashes may be used with caution for a maximum of three consecutive objects, and should not be used in after hyperdashes.
```
Comment (cl8n) [inline wiki/Ranking_Criteria/osu!catch/en.md]:
```suggestion
- Edge dashes may be used with caution for a maximum of three consecutive objects, and should not be used after hyperdashes.
```

File: wiki/Ranking_Criteria/en.md [hunk 6]
```diff
@@ -225,7 +218,7 @@ This category contains explicit allowance statements of concepts and rules that
 - **Acceptable songs with mature lyrics/themes must be marked with an 18+ in the beatmap's description.** This game is for all ages, and so a warning is needed for younger audiences. Some songs, however, are unacceptable for ranking regardless of an 18+ warning. See [song content rules](/wiki/Ranking_Criteria/Song_Content_Rules) for more details.
 - **A beatmapset may only contain one song file used by all difficulties.** Multiple song files within a single beatmap set is unsupported and results in unexpected behaviour with preview times, metadata, etc.
 - **A song's audio file must be of reasonable quality.** Try to find the highest quality source file available rather than ripping a file from a streaming video website. Songs should be normalized to their original release volumes.
-- **A beatmapset's audio file must be no lower than 128kbps and no higher than 192kbps.** Variable bit rate songs must average between that range.
+- **A beatmapset's audio file must be no higher than 192kbps.** Variable bit rate songs must average between that range.
```
Comment (dorothy3242) [inline wiki/Ranking_Criteria/en.md]: "between that range" when there's only a top limit now so there is no range

-> Variable bit rate songs must average beneath this amount.

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 7]
```diff
@@ -121,20 +121,20 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
...
 - **At least 250 ms must be left between objects and the end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 bpm, whereas a 1/2 gap would be required at 120 BPM.
 
 #### Guidelines
 
 - **All distances should be clear on whether they require the player to walk or dash.** This is to ensure that players can easily recognize patterns that require dashing.
-- **Higher-snapped hyperdashes should not be followed by antiflow patterns.** If used, the spacing must not exceed a distance snap of 1.1x the trigger distance and the movement after the hyperdash must be walkable.
-- **Strong hyperdashes should not be used.** For basic hyperdashes, a limit of 1.5x above the trigger distance is recommended. For higher-snapped hyperdashes, a limit of 1.3x is recommended instead.
+- **Higher-snapped hyperdashes should not be followed by antiflow patterns.** If used, the spacing must not exceed a distance snap of 1.1 times the trigger distance and the movement after the hyperdash must be walkable.
```
Comment (osuZiRoX) [inline wiki/Ranking_Criteria/osu!catch/en.md]:
```suggestion
- **Higher-snapped hyperdashes should not be followed by antiflow patterns.** If used, the spacing should not exceed a distance snap of 1.1 times the trigger distance and the movement after the hyperdash must be walkable.
```
I messed up in my latest proposal.

**Comment by Okorin** [comment]
fyi: i approved first and then saw that the other commit he added to have the history from master again changed like 300 files

**Comment by cl8n** [comment]
this looks fine to me, not sure why closed

**Comment by pishifat** [comment]
was advised to make version without that massive commit so did that in #2378

**Comment by cl8n** [comment]
ur advisor is goop

**Pull request closed without merging**